lighthouse-cli/performance-experiment/server.js

```diff
@@ -100,9 +102,10 @@ function rerunRequestHandler(request, response) {
 
       // Add more to flags without changing the original flags
       const flags = Object.assign({}, lhParams.flags, additionalFlags);
-      lighthouse(lhParams.url, flags, lhParams.configs).then(results => {
+      lighthouse(lhParams.url, flags, perfOnlyConfig).then(results => {
```

Comment:
When we rerun tests with configurations (e.g. URL blocking patterns), we only care about performance. So use `perfOnlyConfig` here to get faster result.